### DISCUSSION TITLE
add tar_resources_data.table

### DISCUSSION THREAD

**Discussion by smingerson** [ropensci/targets#697] [Ideas]
Specifically, the threads/percent arguments to `data.table::setDTthreads()`. This will allow specifying the number of threads available for a target separate from the target's code. I'm open to other solutions that allow flexibility between different targets. Still learning all the differences between drake and targets, so might be missing something obvious.

**Comment by wlandau** [comment]
Are you thinking about this for `tar_target(format = "fst_dt")`? If so, you could control that with `fst::threads_fst()` in a project-level `.Rprofile` file. Same goes for `tar_target(format = "feather")` or `"parquet"`, `arrow` lets you control the number of threads using functions at https://arrow.apache.org/docs/r/reference/index.html#section-configuration.
